crates/ursa-pod/Cargo.toml

```diff
@@ -27,6 +27,12 @@ tracing = { version = "0.1", optional = true }
 # Criterion benchmark dependencies
 hyper = { version = "1.0.0-rc.3", features = ["full"], optional = true }
 http-body-util = { version = "0.1.0-rc.2", optional = true }
+quinn = { version = "0.9.3" }
+rcgen = { version = "0.10.0" }
+rustls = { version = "0.20.8", default-features = false, features = ["dangerous_configuration"] }
+s2n-quic = { version = "1.18.0", default-features = false, features = ["provider-address-token-default", "provider-tls-rustls"] }
+# TODO: Add link to issue explaining this.
+zeroize_derive = { version = "<1.4" }
```

Comment:
I think we still want a feature flag `bench-quic` for these and to make the dependencies optional. Atm for example, a user of the library would end up pulling in/compiling all these deps just to use client/server on their own transport. In terms of e2e.rs, it'd be cfg under the flag but additive instead of replacing the entire benchmark. 

For example, with that change I could run all benchmarks with:
```
cargo bench --bench e2e --features bench-hyper --features bench-quic
```